### PR TITLE
Update the example for `cluster_docker_credentials`.

### PR DESCRIPTION
**Pull request by ghost**
## Description
https://jira.mesosphere.com/browse/DCOS-17947

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, and 1.10).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.10/installing/custom/configuration/configuration-parameters.md [hunk 1]
```diff
@@ -121,23 +121,19 @@ Path to a file in OpenSSL PEM format containing the complete CA certification ch
 The dictionary of Docker credentials to pass.
 
 - If unset, a default empty credentials file is created at `/etc/mesosphere/docker_credentials` during DC/OS install. A sysadmin can change credentials as needed. A `systemctl restart dcos-mesos-slave` or `systemctl restart dcos-mesos-slave-public` is required for changes to take effect.
```
Comment (chhsia0) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: The current text seems to indicate that if we don't set anything, the default behavior is that this feature will be enabled with an empty credential file. But AFAIK this is true only if `cluster_docker_credentials_enabled` is set to true. In other words, all docker credential configs described in this section won't have any effect as long as `cluster_docker_credentials_enabled` is unset.

File: 1.9/installing/custom/configuration/configuration-parameters.md [hunk 2]
```diff
@@ -102,25 +102,21 @@ For more information, see the [security documentation](https://docs.mesosphere.c
 The dictionary of Docker credentials to pass. 
 
 - If unset, a default empty credentials file is created at `/etc/mesosphere/docker_credentials` during DC/OS install. A sysadmin can change credentials as needed. A `systemctl restart dcos-mesos-slave` or `systemctl restart dcos-mesos-slave-public` is required for changes to take effect.
```
Comment (chhsia0) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: Ditto.

**Review by chhsia0 — CHANGES_REQUESTED** [review]
@jenafermiller Can you also review this?

File: 1.10/installing/custom/configuration/configuration-parameters.md (hunk 1 above)
Comment (stbof) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: @chhsia0  Please see amended note and separate cluster_docker_credentials_enabled description.

File: 1.9/installing/custom/configuration/configuration-parameters.md (hunk 2 above)
Comment (stbof) [reply 1.9/installing/custom/configuration/configuration-parameters.md]: @chhsia0 Please see amended note and separate cluster_docker_credentials_enabled description.

**Review by chhsia0 — APPROVED** [review]
Ship it!

**Pull request merged**